inc/Integrations/Airtable/AirtableIntegration.php

```diff
@@ -16,8 +16,13 @@ public static function register_blocks(): void {
 			REMOTE_DATA_BLOCKS_AIRTABLE_SERVICE
 		);
 
-		foreach ( $data_source_configs as $config ) {
+		foreach ( $data_source_configs as $config ) {       
 			$data_source = AirtableDataSource::from_array( $config );
+
+			if ( false === $config['service_config']['enable_blocks'] ) {
```

Comment:
You probably want to coalesce to a default value here, this will generate a `WARNING` (sometimes elevated to `ERROR`) if `enable_blocks` is not defined. E.g., if another integration is added or the config is filtered

```suggestion
			if ( false === $config['service_config']['enable_blocks'] ?? true ) {
```